Mod Menu Helper/assets/fo/lang/uk_ua.json

```
@@ -0,0 +1,58 @@
+{
+  "modmenu.dropInfo.line1": "У FO модів є §9✎§r або §8✎§r, альтернативи як §c*§r;",
+  "modmenu.dropInfo.line2": "інші не підтримуються",
+  "modmenu.summaryTranslation.advancementinfo": "Більший, розумніший екран досягнень\n§9✎§8 Кольори, розмір вікна інформації, ...",
+  "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Більший екран досягнень\n§8✎ (без налаштувань)",
+  "modmenu.summaryTranslation.animatica": "Анімовані текстури OptiFine\n§8✎ (лише продвинуті налаштування)",
+  "modmenu.summaryTranslation.antighost": "Виправлення блоків-привидів\n§9✎§8 Клавіши",
```

```suggestion
  "modmenu.summaryTranslation.antighost": "Введіть /ghost, щоб виправити блоки-привиди\n§9✎§8 Клавіши",
```
The first line should be identical to the next string and I confirmed that it fits